Lib/genericpath.py

```diff
@@ -149,7 +149,7 @@ def _check_arg_types(funcname, *args):
         elif isinstance(s, bytes):
             hasbytes = True
         else:
-            raise TypeError('%s() argument must be str or bytes, not %r' %
+            raise TypeError('%s() argument must be str, bytes or os.PathLike object, not %r' %
```

Comment:
```suggestion
            raise TypeError('%s() argument must be str, bytes, or os.PathLike object, not %r' %
```

Reply:
Also, I think we can use an f-string here instead of the older % formatting. We often will avoid refactoring old code *exclusively* for the purpose of using f-strings, but it's preferred for any new code to use f-strings when possible. They're significantly easier to read.